modules/data_integrity_flag/templates/menu_data_integrity_flag.tpl

```diff
@@ -0,0 +1,4 @@
+{*
+  Everything is rendered in index.js now!
+  I'm here because currently Loris breaks without me :(
+*}
```

Comment:
I think it should actually be per-page, not per-module. There are going to be cases where some pages in a module use smarty, and other pages want to be used for things like handling AJAX requests.